crates/common_enums/src/enums.rs

```diff
@@ -3130,3 +3130,50 @@ pub enum OrderFulfillmentTimeOrigin {
     Create,
     Confirm,
 }
+
+/// Indicates how the transaction was authorized by the merchant.
+
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Eq,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    strum::Display,
+    strum::EnumString,
+    ToSchema,
+    Hash,
+)]
+#[router_derive::diesel_enum(storage_type = "text")]
+pub enum SecCode {
+    CCD,
+    POP,
+    PPD,
+    TEL,
+    WEB,
+}
+
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Eq,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    strum::Display,
+    strum::EnumString,
+    ToSchema,
+    Hash,
+)]
+#[router_derive::diesel_enum(storage_type = "text")]
+#[serde(rename_all = "snake_case")]
+#[strum(serialize_all = "snake_case")]
+pub enum AccountType {
```

Comment:
optional - is it possible to re-use `pub enum BankType`? We can extend it with more variants

Reply:
Yes extended BankType